docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -248,6 +248,33 @@
               message: 'Clock on {{ $labels.instance }} is not synchronising. Ensure NTP is configured on this host.',
             },
           },
+          {
+            alert: 'NodeRAIDDegraded',
+            expr: |||
+              'node_md_disks_required - ignoring (state) (node_md_disks{state="active"}) != 0'
```

Comment:
I'm not an expert when it comes to the RAID metrics, but wouldn't it be natural that `node_md_disks_required` can be smaller than `node_md_disks{state="active"}` and this expression would become negative? In other words, shouldn't that be `> 0` in the end?

(And if I'm wrong, it might be worth a comment here to help other not falling into the same trap.)

Reply:
You are right, I missed this